README.md

```diff
@@ -146,6 +146,18 @@ If you want to test the large image feature, where large images can be sent from
 
 This should enable the feature. To confirm the feature has been configured correctly, once Bucketeer is started in the live test mode, visit the Bucketeer status page at: `http://localhost:8888/status`. In the JSON that's returned from the page, you should see that features are enabled and that the `bucketeer.large.images` feature, in particular, is enabled.
 
+## Enable Writing CSVs to a Mounted Filesystem
```

Comment:
Nitpick... `Enabling` since the other headers (e.g., `Including` and `Tweaking` are 'ing words)?

Reply:
:+1: